include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -288,7 +291,7 @@ class Radiation : public ISimulationPlugin
             /*only rank 0 create a file*/
             isMaster = reduce.hasResult(mpi::reduceMethods::Reduce());
 
-            radiation = new GridBuffer<Amplitude, DIM1 > (DataSpace<DIM1 > (elements_amplitude())); //create one int on GPU and host
+            radiation = new GridBuffer<Amplitude, 2> (DataSpace<2>(elements_amplitude(), numTmpResults)); //create one int on GPU and host
```

Comment:
note: I explicitly uses `2` instead of `DIM2` to reflect that it is not representing the special dimension.

Reply:
Probably the comment in this line needs to be updated, since it's not one any more?